websocket/connection.go

```diff
@@ -130,6 +137,21 @@ func NewDurableConnection(target string, messageChan chan []byte, logger *zap.Su
 		}
 	}()
 
+	// Keep sending pings 3 times per pongTimeout interval.
+	go func() {
+		ticker := time.NewTicker(pongTimeout / 3)
```

Comment:
they don't divide evenly 😞

Reply:
Does it matter that much? 😂

Reply:
Well, it's a loss of 10% (1sec) of precision :) If the timeout was defined in ms it'd just 1ms, though :)